app/file.go

```diff
@@ -36,7 +36,7 @@ import (
 	"github.com/mattermost/mattermost-server/v5/mlog"
 	"github.com/mattermost/mattermost-server/v5/model"
 	"github.com/mattermost/mattermost-server/v5/plugin"
-	"github.com/mattermost/mattermost-server/v5/services/filesstore"
+	"github.com/mattermost/mattermost-server/v5/shared/filesstore"
```

Comment:
Just a small nit from my side: can we use this opportunity to rename it to `filestore` rather than `filesstore`?

Reply:
Done.